src/realm/obj.cpp

```diff
@@ -1089,6 +1187,49 @@ Obj& Obj::set<ObjKey>(ColKey col_key, ObjKey target_key, bool is_default)
     return *this;
 }
 
+template <>
+Obj& Obj::set<ObjLink>(ColKey col_key, ObjLink target_link, bool is_default)
```

Comment:
Seems as if a lot of code could be shared between this method and set\<Mixed\>

Reply:
Not more than between all set operations, I think.